src/Riok.Mapperly/Descriptors/MappingBuilders/EnumMappingBuilder.cs

```diff
@@ -83,6 +83,18 @@ private static TypeMapping BuildNameMapping(MappingBuilderContext ctx, bool igno
                 ctx.Target);
         }
 
+        var targetMembers = ctx.Target
+            .GetMembers()
+            .OfType<IFieldSymbol>();
+
+        if (enumMemberMappings.Count < targetMembers.Count())
```

Comment:
This does not really detect all unmapped items. Eg. if the source enum contains the values `Value1, Value2, Value3` and the target enum contains the values `Value10`, `Value20`, `Value30` comparing the count does not work. It should work ByValue, ByName and also take the case sensitivity settings into account.

Reply:
I changed the implementation to look for missing values in source & target enums. All variations "ByValue, ByName and case sensitivity" are taken into account.